library/flink

```diff
@@ -3,27 +3,32 @@
 Maintainers: The Apache Flink Project <dev@flink.apache.org> (@ApacheFlink)
 GitRepo: https://github.com/apache/flink-docker.git
 
-Tags: 2.0-preview1-scala_2.12-java17, 2.0-scala_2.12-java17, 2.0-preview1-java17, 2.0-java17
+Tags: 2.0.0-scala_2.12-java21, 2.0-scala_2.12-java21, scala_2.12-java21, 2.0.0-java21, 2.0-java21, java21
```

Comment:
We support jdk21 in Flink 2.0, this after the preview release, so preview1 doesn't include it.